docs/source/guides/orchestrator_transcoder_gpu.rst

```diff
@@ -0,0 +1,115 @@
+How To Run An Orchestrator With GPU Transcoding
+===============================================
+
+Orchestrators can use a GPU to transcode streams.
```

Comment:
Should we use "can use GPUs to transcode streams" to not give the idea of being limited to a single GPU?

Reply:
Fixed in 34ecfc0